data/event-schemas/schema/m.room.create.yaml

```diff
@@ -6,7 +6,9 @@ properties:
   content:
     properties:
       creator:
```

Comment:
needs removing from the `required` list.

Reply:
Also applied the same for `redacts`.